single_include/catch2/catch.hpp

```diff
@@ -17531,7 +17531,7 @@ namespace Catch {
 extern "C" int wmain (int argc, wchar_t * argv[], wchar_t * []) {
 #else
 // Standard C/C++ main entry point
-int main (int argc, char * argv[]) {
+int __cdecl main (int argc, char * argv[]) {
```

Comment:
No changes to `single_include` as part of PRs!
```suggestion
int main (int argc, char * argv[]) {
```